py21cmsense/sensitivity.py

```diff
@@ -474,3 +485,71 @@ def plot_sense_2d(self, sense2d):
         cbar.set_label(r"$\log_{10} \delta \Delta^2$ [mK^2]", fontsize=14)
         plt.xlabel(r"$k_\perp$ [h/Mpc]", fontsize=14)
         plt.ylabel(r"$k_{||}$ [h/Mpc]", fontsize=14)
+
+    def write(
+        self,
+        filename: str | Path,
+        thermal: bool = True,
+        sample: bool = True,
+        prefix: str = None,
+    ) -> str | Path:
```

Comment:
This function doesn't actually return anything as it is currently written but you notate that it can return either a str or a Path object.

Reply:
woops!

Reply:
OK fixed.